packages/code-studio/src/settings/FormattingSectionContent.jsx

```diff
@@ -718,6 +780,54 @@ export class FormattingSectionContent extends PureComponent {
               </Checkbox>
             </div>
           </div>
+          <div className="form-row mb-2">
+            <label className="col-form-label col-3">Decimal</label>
+            <div className="col-9">
+              <input
+                className={classNames(
+                  'form-control',
+                  'flex-grow-1',
+                  'default-decimal-format-input',
+                  {
+                    'is-invalid': !DecimalColumnFormatter.isValid(
+                      DecimalColumnFormatter.makeCustomFormat(
+                        defaultDecimalFormatString
+                      )
+                    ),
+                  }
+                )}
+                data-lpignore
+                placeholder={DecimalColumnFormatter.DEFAULT_FORMAT_STRING}
+                type="text"
+                value={defaultDecimalFormatString}
+                onChange={this.handleDefaultDecimalFormatChange}
+              />
+            </div>
+          </div>
+          <div className="form-row mb-2">
```

Comment:
```suggestion
          <div className="form-row mb-3">
```
And change `mb-3` to `mb-2` on line 773.